docs/content/basic/installing_for_developers.rst

```diff
@@ -78,40 +80,29 @@ repository, have a look at the article: `A successful git branching model
 Setting up your environment
 ---------------------------
 
-So that you can access SimPEG from anywhere on your computer, you need to add
-it to your path. This can be done using symlinks. In your :code:`git` directory,
-create a directory called :code:`python_symlinks`.
-
-.. image:: ../../images/gitfolders.png
-    :align: center
-    :width: 40%
-
-Open a terminal in this directory and create a symlink for SimPEG ::
-
-    ln -s ../SimPEG/SimPEG .
-
-Then, in your shell, you need to add a :code:`PYTHONPATH` variable. For Mac and
-Linux, if you are using Z shell (`Oh My Zsh <http://ohmyz.sh/>`_ is used by a
-lot of SimPEG developers) or bash open the config in a text editor, ie::
-
-    nano ~/.zshrc
+To get started developing SimPEG we recommend setting up an environment with Anaconda_
```

Comment:
Again, I think we should probably not push on "anaconda" - in the end, our codes are not even available on anaconda, they are available on conda-forge. We should strengthen conda-forge (mambaforge).